lightstep/recorder.py

```diff
@@ -184,23 +230,20 @@ def _construct_report_request(self):
         report = None
         with self._mutex:
             report = ttypes.ReportRequest(self._runtime, self._span_records,
-                                          self._log_records)
+                                          None)
             self._span_records = []
             self._log_records = []
-
-        for log in report.log_records:
-            if log.stable_name is not None:
-                log.stable_name = str(log.stable_name)
-            if log.payload_json is not None:
-                try:
-                    log.payload_json = \
-                                       jsonpickle.encode(log.payload_json,
-                                                         unpicklable=False,
-                                                         make_refs=False,
-                                                         max_depth=constants.JSON_MAX_DEPTH)
-                except:
-                    log.payload_json = jsonpickle.encode(constants.JSON_FAIL)
-                
+        for span in report.span_records:
+            for log in span.log_records:
+                if log.payload_json is not None:
+                    try:
+                        log.payload_json = \
+                                           jsonpickle.encode(log.payload_json,
+                                                             unpicklable=False,
+                                                             make_refs=False,
+                                                             max_depth=constants.JSON_MAX_DEPTH)
+                    except:
+                        log.payload_json = jsonpickle.encode(constants.JSON_FAIL)
         return report
 
     def _add_log(self, log):
```

Comment:
But don't we still want to populate the `ttypes.ReportRequest.log_request` field? Also since we're getting rid of the `Runtime._log_records` field then we no longer need the `_add_log()` method, no?